cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4CareGapsBundleBuilder.java

```diff
@@ -98,7 +98,8 @@ public List<Parameters.ParametersParameterComponent> makePatientBundles(
             @Nullable ZonedDateTime periodEnd,
             List<String> subjects,
             List<String> statuses,
-            List<IdType> measureIds) {
+            List<IdType> measureIds,
+            boolean isDocumentMode) {
```

Comment:
Why not expose both makePatientBundle() methods as makePatientBundleNonDocumentMode() and makePatientBundleDocumentMode(), make the other methods private, and only pass the boolean internally?   This will be a clearer contract for API consumers.

Reply:
attempted this initially, but ended up having too much duplicate code, this seemed more efficient